src/service/impl/stops/departure-time.ts

```diff
@@ -0,0 +1,182 @@
+import client from '../graphql-client';
+import query from './departure-time.graphql';
+import {
+  APIError,
+  DeparturesRealtimeData,
+  DepartureRealtimeData,
+  DepartureRealtimeQuery
+} from '../../types';
+import { Result } from '@badrap/result';
+
+type DepartureRealtimeDataFromGraphQL = {
```

Comment:
At some point I'll look into integrating type gen from GraphQL queries. But manually for now